content/_index.md

```diff
@@ -2,9 +2,39 @@
 title: "🍲 Based Cooking 🍳"
 ---
 
-## Categories
-
-{{< tagcloud >}}
+## What do you want to cook?
+
+<div class="search">
+  <input type="text" id="search" placeholder="Search...">
+  <button class="clear-search">
+    <svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Backspace</title><path d="M135.19 390.14a28.79 28.79 0 0021.68 9.86h246.26A29 29 0 00432 371.13V140.87A29 29 0 00403.13 112H156.87a28.84 28.84 0 00-21.67 9.84v0L46.33 256l88.86 134.11z" fill="none" stroke="currentColor" stroke-linejoin="round" stroke-width="32"></path><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32" d="M336.67 192.33L206.66 322.34M336.67 322.34L206.66 192.33M336.67 192.33L206.66 322.34M336.67 322.34L206.66 192.33"></path></svg>
+  </button>
+</div>
+
+<script>
+document.addEventListener('DOMContentLoaded', () => {
+  const rec = document.querySelectorAll('#artlist li')
```

Comment:
```suggestion
  const ul = document.querySelector('#artlist')
  const rec = document.querySelectorAll('#artlist li')
```

Reply:
New Selector for the `ul`.

Reply:
I commit while you were writing this. Is my solution for getting the article list acceptable? Not sure what's computationally efficient or not in JS lol

Reply:
The way you've done it is fine. I'm not sure which is better but If anything it's probably yours anyways.